frontend/components/messenger/message_list_item.jsx

```diff
@@ -38,13 +38,13 @@ class MessageListItem extends React.Component {
   }
 
   render() {
-    const { body, created_at, id } = this.props.message;
+    const { body, created_at, id, user } = this.props.message;
```

Comment:
at the top of this file you have a commented out line, between import statements and class definition